content/blog/2024/03/2024-03-14-why-matrix-org.md

```diff
@@ -73,6 +73,6 @@ There are alternatives to a prominent onboarding homeserver like the Matrix.org
 * Portable identities are very desirable but will take time to materialise.
 * P2P Matrix depends on a portable identity mechanism and is further away in the future.
 
-As you can see, we at the Foundation need to strike a balance as we pursue goals that are occasionally in tension with one another, and the Matrix.org homeserver is a prime example. We are committed both to making Matrix more accessible, and to doing the work to de-center the Matrix.org homeserver.
+As you can see, we at the Foundation need to strike a balance as we pursue goals that are occasionally in tension with one another, and the Matrix.org homeserver is a prime example. We are committed both to making Matrix more accessible, and to doing the work to decenter the Matrix.org homeserver.
```

Comment:
Is this actually correct? :D I am not english native either but I would have thought `de-center` makes sense in context here.

Reply:
https://www.merriam-webster.com/dictionary/decenter